ql/experimental/termstructures/crosscurrencyratehelpers.hpp

```diff
@@ -46,7 +46,9 @@ namespace QuantLib {
                                              ext::shared_ptr<IborIndex> quoteCurrencyIndex,
                                              Handle<YieldTermStructure> collateralCurve,
                                              bool isFxBaseCurrencyCollateralCurrency,
-                                             bool isBasisOnFxBaseCurrencyLeg);
+                                             bool isBasisOnFxBaseCurrencyLeg,
+                                             Frequency paymentFrequency,
+                                             Integer paymentLag);
```

Comment:
I would add the defaults here too, on the off chance that someone inherited from this class.